routes/userRoute.js

```diff
@@ -0,0 +1,50 @@
+const
+_      = require('lodash'),
+User   = require('../models/User'),
+utils  = require('../modules/utils');
+
+function list(params, callback) {
+    callback({ message: 'not allowed' });
+}
+
+function get(params, callback) {
+    if (params.id) {
+        User.findOne({ _id: params.id }, utils.send(callback));
+    } else {
+        utils.send(callback);
```

Comment:
I guess the callback is not called here, right? utils.send just returns another function.